indico/modules/rb_new/operations/rooms.py

```diff
@@ -75,6 +79,20 @@ def _query_managed_rooms(user):
     return Room.query.filter(Room.is_active, db.or_(Room.owner == user, is_manager))
 
 
+def _make_detailed_room(room):
```

Comment:
I'm not sure this belongs in `operations`. It doesn't create anything on the DB and it's more of a `util` function, I believe. `_make_detailed_room` is also a bit of a misleading name. I'd call it `create_room_dict` or similar. Please also note that if this function is to be imported from outside the source file (which is the case), you'd normally remove the underscore (as per common practice, not sure if there's a written rule for it).